.github/workflows/PTF-TEST.yml

```diff
@@ -0,0 +1,100 @@
+name: "PTF-backend"
+# This is just a temporary workflow to test the PTF backend
+# Will be merged to ci-dpdk-ptf-tests.yml once it is ready
+on:
+  push:
+    branches:
+      - ptf-test-pr
```

Comment:
This test needs to be updated. I would recommend replacing the existing Github action workflow.

Reply:
yes, it will replace the existing dpdk ptf ci.